proguard/proguard-moshi.pro

```diff
@@ -0,0 +1,10 @@
+-dontwarn okio.**
+-dontwarn javax.annotation.**
+-keepclasseswithmembers class * {
+    @com.squareup.moshi.* <methods>;
+}
+-keep @com.squareup.moshi.JsonQualifier interface *
+
+-keepclassmembers class kotlin.Metadata {
```

Comment:
DroidKaigi 2018 app doesn't use Kotlin-Moshi library(it rule for kotlin reflection library)
i seems unnecessary rule 🐰